tests/acceptance/test_project_settings_sampling.py

```diff
@@ -142,7 +142,10 @@ def test_add_uniform_rule_with_recommended_sampling_values(self):
                 key="sentry:dynamic_sampling", project=self.project
             )
             saved_sampling_setting = project_option.value
-            serializer = DynamicSamplingSerializer(data=saved_sampling_setting)
+            serializer = DynamicSamplingSerializer(
+                data=saved_sampling_setting,
+                context={"project": self.project, "request": self.make_request(user=self.user)},
```

Comment:
funny, I worked on the exact same code today and used a slightly different solution https://github.com/getsentry/sentry/blob/53731165e168285fc859b0e77043636e067a1f92/tests/acceptance/test_project_settings_sampling.py#L91-L92

Reply:
Which one do you like more? I decided to use a pre-made fixture for consistency reasons.

Reply:
I don't have any preferences :)